data/mods/CrazyCataclysm/crazy_monsters.json

```diff
@@ -14,6 +14,28 @@
     "copy-from": "mon_zombie_electric",
     "special_attacks": [ [ "SHOCKING_REVEAL", 25 ] ]
   },
+  {
+    "id": "debug_mon",
```

Comment:
Debug monster is a step too far here, it's not just silly, it's explicitly 4th wall breaking.

Reply:
So I remove it?